fdbbackup/backup.actor.cpp

```diff
@@ -697,6 +697,9 @@ static void printAgentUsage(bool devhelp) {
 		   "  --logdir PATH  Specifes the output directory for trace files. If\n"
 		   "                 unspecified, defaults to the current directory. Has\n"
 		   "                 no effect unless --log is specified.\n");
+	printf("  --trace_format FORMAT\n"
```

Comment:
Tab/space usage is inconsistent with surrounding lines.

Reply:
Done